rdb_test.go

```diff
@@ -0,0 +1,64 @@
+//go:build rocksdb
+// +build rocksdb
+
+package db
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRDBNewDB(t *testing.T) {
+	name := fmt.Sprintf("test_%x", randStr(12))
+	dir := os.TempDir()
+	db, err := NewDB(name, RDBBackend, dir)
+	defer cleanupDBDir(dir, name)
+	require.NoError(t, err)
+
+	_, ok := db.(*RDB)
+	assert.True(t, ok)
+}
+
+func TestRDBStats(t *testing.T) {
+	name := fmt.Sprintf("test_%x", randStr(12))
+	dir := os.TempDir()
+	db, err := NewDB(name, RDBBackend, dir)
+	defer cleanupDBDir(dir, name)
+	require.NoError(t, err)
```

Comment:
Is `defer` in the correct order? I saw almost same codes, but `defer` follows `require.NoError()`.
If the reader (human) does not know the internals of `NewDB()` and `cleanupDBDir()`, using `os.RemoveAll()` right after `os.TempDir()` could be a better choice. I saw it in many places in this PR.
If you accept my opinion, I also ask you to fix all the similar lines in this PR.